src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -387,6 +387,16 @@ export class ExtensionEditor extends BaseEditor {
 			template.license.onclick = null;
 			template.license.style.display = 'none';
 		}
+
+		// copied from the the extension.url condition block above
```

Comment:
What about the license link? And if we're going to do this I'd say you should put these in an else so that we don't risk something changing and there being two onclick handlers.

Reply:
Oh we already add that right above. Still think they should go into an else though. 

---
In reply to: [343930531](https://github.com/microsoft/azuredatastudio/pull/8271#discussion_r343930531) [](ancestors = 343930531)

Reply:
And it'd be good to update the license code to be the same as the vscode stuff - looks like it's diverged a bit from the original implementation. 

---
In reply to: [343930890](https://github.com/microsoft/azuredatastudio/pull/8271#discussion_r343930890) [](ancestors = 343930890,343930531)